src/server/routes/job.ts

```diff
@@ -0,0 +1,24 @@
+import express from "express";
+
+import { asyncHandler } from "../utils";
+import { BaseJob } from "../../types";
+import { ScheduleHandler } from "../jobs";
+
+export const router = express.Router();
+
+router.post(
+    "/create",
+    asyncHandler<{}, {}, BaseJob>(async (req, res) => {
+        console.log("Job scheduled", new Date(req.body.jobDate).toISOString());
+        console.log("Current time is", new Date().toISOString());
+        console.log(
```

Comment:
Are these for testing? Make a note to remove these once we can confirm this works properly.

Reply:
I use `console.log` as a logger, maybe we could get a dedicated one later. Anyways they are just info logs for paper trail, our log management service.